### PR TITLE
Rename CRT to Threaded Discussions

### PR DESCRIPTION
**Pull request by marianunez**
#### Summary

Rename Collapsed Reply Threads to Threaded Discussions in the Admin Console screens. No changes are done to config settings to avoid migration changes. 

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-57431

#### Screenshots

|  before  |  after  |
|----|----|
| Screenshot 2024-07-25 at 1 54 20 PM | Screenshot 2024-07-25 at 2 15 48 PM |

#### Release Note

```release-note
Rename Collapsed Reply Threads to Threaded Discussions in System Console
```

### CODE REVIEW

**Review by esethna — APPROVED** [review]

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
We have the term CRT even in the user settings as well. Dont we want to change there as well?

**Comment by cwarnermm** [comment]
I agree with @M-ZubairAhmed that we have CRT references in user preferences that also need to be renamed. 

There's also the 2 modal popups about CRT threads (one for admins and one for end users); not sure how/when the end user one is triggered, but worth reviewing/updating.

Screenshot 2024-07-26 at 8 43 40 AM

Screenshot 2024-07-26 at 8 42 59 AM

**Comment by marianunez** [comment]
> We have the term CRT even in the user settings as well. Dont we want to change there as well?

Yes, I had changed the title in the translation file for the user settings. Just missed the Default Message. Added

File: webapp/channels/src/components/admin_console/admin_definition.tsx [hunk 1]
```diff
@@ -2515,16 +2515,16 @@ const AdminDefinition: AdminDefinitionType = {
...
-                            label: defineMessage({id: 'admin.experimental.collapsedThreads.title', defaultMessage: 'Collapsed Reply Threads'}),
-                            help_text: defineMessage({id: 'admin.experimental.collapsedThreads.desc', defaultMessage: 'When enabled (default off), users must enable collapsed reply threads in Settings. When disabled, users cannot access Collapsed Reply Threads. Please review our <linkKnownIssues>documentation for known issues</linkKnownIssues> and help provide feedback in our <linkCommunityChannel>Community Channel</linkCommunityChannel>.'}),
+                            label: defineMessage({id: 'admin.experimental.collapsedThreads.title', defaultMessage: 'Threaded Discussions'}),
+                            help_text: defineMessage({id: 'admin.experimental.collapsedThreads.desc', defaultMessage: 'When enabled (default off), users must enable Threaded Discussions in Settings. When disabled, users cannot access Threaded Discussions. Please review our <linkKnownIssues>documentation for known issues</linkKnownIssues> and help provide feedback in our <linkCommunityChannel>Community Channel</linkCommunityChannel>.'}),
```
Comment (marianunez) [inline webapp/channels/src/components/admin_console/admin_definition.tsx:2527]: @esethna do we still add this link to known issues for CRT in the System Console now that is no longer in Beta?

**Comment by marianunez** [comment]
> There's also the 2 modal feature discovery popups about CRT threads (one for admins and one for end users); not sure how/when the end user one is triggered, but worth reviewing/updating.

@cwarnermm those modals were product notices that we push out from a different repository and are not part of the product. Those are no longer live so no additional changes required for them.

**Comment by cwarnermm** [comment]
@marianunez - I am able to make the admin modal show up in my cloud instance by setting CRT to off by default. Would that one be part of the product and something to update?

**Comment by esethna** [comment]
Also realized, there is a CRT end user tutorial tour for first time users. Not sure if that cas any references to "Collapsed Reply Threads"? @marianunez

**Comment by amyblais** [comment]
> can you help update the term "Collapsed Reply Threads" > "Threaded Discussions" in any notices we are currently shipping?

Opened https://github.com/mattermost/notices/pull/410.

**Comment by marianunez** [comment]
> Also realized, there is a CRT end user tutorial tour for first time users. Not sure if that has any references to "Collapsed Reply Threads"? @marianunez

@esethna Double checked and none of the wording in the Tutorial mentions the feature `Collapsed Reply Threads or CRT`. The tutorial references the new areas of `Threads`, `Follow`, `Unreads`, etc.

**Review by M-ZubairAhmed — APPROVED** [review]
Please also cross check the E2Es as well

**Comment by marianunez** [comment]
> Please also cross check the E2Es as well

Double checked the E2E tests and there is no action to change CRT settings or any action that uses the UI elements. All tests are using the config directly that was not impacted with this change. 

Thanks @M-ZubairAhmed!

**Comment by amyblais** [comment]
@cwarnermm Kind reminder for this PR review.

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**